sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/DateTimeUtilsSuite.scala

```diff
@@ -496,8 +496,8 @@ class DateTimeUtilsSuite extends SparkFunSuite with Matchers with SQLHelper {
       test("2016-03-13 02:00:00", tz, "2016-03-13 10:00:00.0")
       test("2016-03-13 03:00:00", tz, "2016-03-13 10:00:00.0")
       test("2016-11-06 00:59:59", tz, "2016-11-06 07:59:59.0")
-      test("2016-11-06 01:00:00", tz, "2016-11-06 08:00:00.0")
-      test("2016-11-06 01:59:59", tz, "2016-11-06 08:59:59.0")
+      test("2016-11-06 01:00:00", tz, "2016-11-06 09:00:00.0")
```

Comment:
<img width="797" alt="Screenshot 2020-04-11 at 19 01 34" src="https://user-images.githubusercontent.com/1580697/79048599-f046e180-7c26-11ea-9940-eddee9785fb4.png">

Reply:
what's the behavior of Spark 2.4?

Reply:
See the same test in 2.4 https://github.com/apache/spark/blob/branch-2.4/sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/DateTimeUtilsSuite.scala#L580-L581 but it sets PST time zone.

Reply:
as we have looked into the JDK code before, PST is treated as LA. So the behavior is the same as 2.4, and it's great!

can we highlight it in the `Does this PR introduce any user-facing change?` section? Say it's changed back to the 2.4 behavior.

Reply:
done